### DISCUSSION TITLE
Can i get unlimited alerts with this?

### DISCUSSION THREAD

**Discussion by Ishwar7747** [soranoo/TradingView-Free-Webhook-Alerts#10] [General]
Can i get unlimited alerts with this program?

And do i have to keep my pc always on for it?

**Comment by soranoo** [comment]
Concerning your first question, you cannot get unlimited alerts with this program because the number of alerts is limited by TradingView. Here is the detail,
- **1** alert for the **BASIC** account
- **20** alerts for the **PRO** account
- **100** alerts for the **PRO+** account
- **400** alerts for the **PREMIUM** account
(ref: https://www.tradingview.com/gopro/#compare)

This program is mainly for TradingView free plan users who want to use webhooks.

Regarding your second question, you should keep the program and your PC on when you are using the program.

**Comment by Ishwar7747** [comment]
Ok thanks

Do u know any pine script trick or something for unlimited alerts

**Comment by soranoo** [comment]
Alert is offered by TradingView, I don't think we can use any pine script or any third-party method to get unlimited alerts.
